bot/main.py

```diff
@@ -95,15 +107,23 @@ def game(message):
     Gameday.games.append(Game(*parsed))
     bot.send_message(message.chat.id, 'Saved! Use /info')
 
+
 @bot.message_handler(commands=['info'])
 @guard()
 def info(message):
     bot.send_message(message.chat.id, Gameday.getInfo(), parse_mode='markdown')
 
+
 @bot.message_handler(commands=['end'])
 @guard(admins)
 def end(message):
-    bot.reply_to(message, 'TODO: call @klicunou code')
+
+    games_csv_filepath = os.path.join(REPO_ROOT_DIR, 'data/games.csv')
+    games = pd.read_csv(games_csv_filepath)
```

Comment:
why you need to read `games` here?

Reply:
could you just [`append`](https://www.guru99.com/reading-and-writing-files-in-python.html#2) new games to file?

Reply:
I have to calculate a game index, so I need to know previous indexes.